designs/octoplushy/src/head.mjs

```diff
@@ -613,15 +613,16 @@ const options = {
     min: 5,
     max: 500,
     menu: 'style',
+    // eslint-disable-next-line no-unused-vars
     toAbs: (val, { options }, mergedOptions) =>
       ((mergedOptions.sizeConstant * val * 2) / 3.1415) *
-      (options.type === undefined
+      (mergedOptions.type === undefined
```

Comment:
Under which circumstances is the type undefined? Also wouldn't that be already handled by the `: 1` at the end?

Reply:
Yes, that should not happen. Removed